deploy/001_deploy_hattimelockcontroller.js

```diff
@@ -0,0 +1,46 @@
+const CONFIG = require("../config.json");
+const { network } = require("hardhat");
+const TIMELOCK_ADMIN_ROLE = "0x5f58e3a2316349923ce3780f8d587db2d72378aed66a8261c916544fa6846ca5";
+
+const func = async function (hre) {
+  const config = CONFIG[network.name];
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy, execute } = deployments;
+
+  const { deployer } = await getNamedAccounts();
+  
+  let governance = config["governance"];
+  if (!governance && network.name === "hardhat") {
+    governance = deployer;
+  }
+
+  let hatGovernanceDelay;
+  if (config["timelockDelay"]) {
```

Comment:
I think this is not a good pattern, it makes the config files really hard to understand if there are default values with complex logic hidden in deployment scripts.  Better make it obligatory to have all the settings in set explcitly in the config file and fail otherwise (if we must, we can have a `defaults` sectino in the config file as well)